### ISSUE TITLE
Cannot use -watch with file input

### ISSUE THREAD

**Issue by cisso** [dailymuse/gosass#6]
When using the -watch argument, gosass requires -input to be a directory and will fail if -input references a file.
It would be lovely if we could use -watch for files as well. :)

**Comment by ysimonson** [comment]
Agreed! Unfortunately I can't get to it in the near-future, but PRs would be happily accepted.
